crates/optimism/node/src/node.rs

```diff
@@ -721,6 +724,19 @@ impl OpNetworkBuilder {
     }
 }
 
+pub fn get_all_op_bootnodes<Node>(ctx: &BuilderContext<Node>) -> Vec<Bootnode>
+where
+    Node: FullNodeTypes<Types: NodeTypes<ChainSpec: Hardforks>>,
+{
+    let network_name = ctx.config().chain.chain.clone().to_lowercase();
+
+    if network_name.contains("mainnet") {
+        params::V5OPBootnodes.to_vec()
+    } else {
+        params::V5OPTestnetBootnodes.to_vec()
```

Comment:
actually we can make this redundant

if we change just change this to `is_optimism()` and `is_testnet()`:

https://github.com/paradigmxyz/reth/blob/81cff4a90b00cd5fbbb1ee122611266c14e86ed4/crates/chainspec/src/spec.rs#L646-L647